ambry-store/src/main/java/com.github.ambry.store/CompactionLog.java

```diff
@@ -313,13 +327,13 @@ static CycleLog fromBytes(DataInputStream stream, StoreKeyFactory storeKeyFactor
      */
     Phase getPhase() {
       Phase phase;
```

Comment:
I think `Phase` is a derived attribute and is a helper for recovery (and for determining when compaction is done). So it is not necessary to store it or set it explicitly. Start times of phases can never go back to being `UNINITIALIZED` once set and the function works under this rule.